templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +24,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                  <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">UPDATE</a>
```

Comment:
Ensure that the URL pattern `taxi:manufacturer-update` is correctly defined in your `urls.py` file, as it is used here to link to the manufacturer update page.